src/app/workspace/resource/values/link-value/link-value.component.ts

```diff
@@ -114,10 +121,42 @@ export class LinkValueComponent extends BaseValueDirective implements OnInit, On
     ngOnInit() {
         const linkType = this.parentResource.getLinkPropertyIriFromLinkValuePropertyIri(this.propIri);
         this.restrictToResourceClass = this.parentResource.entityInfo.properties[linkType].objectType;
+        // console.log('parentResource: ', this.parentResource);
+        // console.log('linkType: ', linkType);
+        // console.log('restrictToResourceClass: ', this.restrictToResourceClass);
+        // console.log('resourceClasses: ', this.resourceClasses);
+        // console.log('ontoIri: ', this.ontoIri);
+        // console.log('propIri: ', this.propIri);
```

Comment:
can we remove these commented console logs?

Reply:
remove all the console logs in [a61671a](https://github.com/dasch-swiss/dsp-app/pull/686/commits/a61671ada5ad610fe9d7c6343685aa22ad598536) 🙈